app/assets/javascripts/application.js

```diff
@@ -13,3 +13,51 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+$(document).ready(function () {
```

Comment:
I think it would be better to move this code out of application.js into its separate file (the same way as we do in caseflow certification) @nemodjuric